### DISCUSSION TITLE
Q&A: Mongodb

### DISCUSSION THREAD

**Discussion by whxyu1337** [IgorKowalczyk/majobot#36] [Q&A] [answered]
Since many user having error with mysql database,why don't u make a verson using mongoDB instead?,it's much easier

**Comment by IgorKowalczyk** [comment, marked as answer]
I am currently thinking about 2 options
1. Custom databases (save to MYSQL, mongodb or json file). You will be able to change this in Majo.exe configuration
2. Switch completely to mongodb

**Comment by whxyu1337** [comment]
i think u should choose option 2 bc it easier for you

**Comment by limefocus** [comment]
I agree completely shifting to mongodb is more easier and also removed the need for a lot of code if we host the db using mongodb atlas for one of the plan it is only 0.10/million requests (not sure it’s what I remember)
